onnxruntime/core/providers/cuda/cuda_provider_options.h

```diff
@@ -16,6 +16,9 @@ struct CudaProviderOptions {
   // use cuda device with id=0 as default device.
   OrtDevice::DeviceId device_id = 0;
 
+  //set cudnn conv algo search option
+  int cudnn_conv_algo_search = 0;
```

Comment:
>0 [](start = 31, length = 1)

By the way, I think you have a reason to use numbers(0, 1, 2) instead of enum class or constant. Is it because there is no good place to define them?